### DISCUSSION TITLE
Ultima VII Soundtrack Remake

### DISCUSSION THREAD

**Discussion by JDHardingMusic** [exult/exult#313] [Show and tell]
**Note: I lost access to my original forum account ages ago. So I created a new one for Github.**

**This is my soundtrack remake, which I posted on the original Exult forum. Some links have changed since then.**

**All music was produced in Reason with Kontakt as the main VST plugin. Various instrument libraries were used, but I can't remember which ones. These are not MIDI files and do not use any soundfonts.**

**I do not offer any technical support. Exult has plenty of documentation on how to use the OGG files within the game.**

**[Bandcamp Album](https://jdharding.bandcamp.com/album/ultima-vii)**
**[Free Exult-Ready OGG File Download](https://mega.nz/folder/k6xSEYxB#B1EZfbEGCPeX6Vy9HN2LpA)**

**Comment by drcode1** [comment]
Those sound really good!

**Comment by DominusExult** [comment]
thanks for the new links to your awesome remake.
Yes, sorry, the forum first had to be converted from Phorum to phpBB which resulted in a reset of all passwords. And eventually the deprecation of SourceForge support brought us (especially me) to retire the forum in favor of Github Discussions. Not quite the same but more future proof.
